plugins/tracker-resources/src/components/issues/Issues.svelte

```diff
@@ -30,6 +31,8 @@
     ['active', tracker.string.Active, {}],
     ['backlog', tracker.string.Backlog, {}]
   ]
+  let [[mode]] = config
+  const loc = parseLocation(new URL(`${window.location.href}/${mode}`))
```

Comment:
Should we use a #fragment for mode? @BykhovDenis

Reply:
Also I think we need to pass extra path from plugins/workbench-resources/src/components/Workbench.svelte to individual components and pass back options to put into path, instead of adding into each individual component